core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -869,7 +869,7 @@ public void listStatus(AlluxioURI path, ListStatusContext context,
       auditContext.setSrcInode(inode);
       DescendantType descendantTypeForListStatus =
           (context.getOptions().getRecursive()) ? DescendantType.ALL : DescendantType.ONE;
-      listStatusInternal(rpcContext, inodePath, auditContext, descendantTypeForListStatus,
+      listStatusInternal(context, rpcContext, inodePath, auditContext, descendantTypeForListStatus,
```

Comment:
in general, why is the `isCancelled()` not part of the rpcContext? It would seem appropriate as part of an rpc context.

Reply:
`RpcContext` is a master side construct. I'd still need to pass `CallTracker` into FSM implementations then I could offload them into `RpcContext`. Since we already pass around op-contexts in most of the code, I am just calling them from there. This is the only part where we were not passing the op-context.

Reply:
I see. I thought it made sense for something called `RPCContext` to have a method like `isCancelled`, but we can always change it later.